docs/source/download.rst

```diff
@@ -1,64 +1,107 @@
+
 .. code-block:: shell
 
-   # get the code
-   git clone https://github.com/devitocodes/devito.git
-   cd devito
+   # NOTE (admin-sudo access may be needed) for some steps
 
-   # 1. run the tests
-   docker-compose run devito /tests
+   # 1. Pull Devito image
+   docker pull devitocodes/devito:cpu-latest
 
-   # 2. start a jupyter notebook server on port 8888
-   docker-compose up devito
+   # 2. (Optional but recommended) Run the tests
+   docker run --rm --name testrun 'devitocodes/devito:cpu-latest' pytest tests/test_operator.py
 
-   # 3. start a bash shell with devito
-   docker-compose run devito /bin/bash
+   # 3. Start a bash shell with Devito
+   docker run --rm -it devitocodes/devito:cpu-latest /bin/bash
 
-The three sample commands above have only been included as illustrations of typical
-uses of Devito inside a container. They are not required to be run in that order.
+   # 4. Start a Jupyter notebook server on port 8888
+   docker run --rm -it -p 8888:8888 devitocodes/devito:cpu-latest
 
-1. Command 1 above runs the unit tests included with Devito to check whether the
-installation succeeded. This is not necessary but a handy first thing to run. As
-long as the first few tests are passing, it is possible to press `Ctrl+C` to stop
-the testing and continue.
+Commands 2, 3 and 4 above are illustrations of typical uses of Devito inside a container.
+They are not required to be run in that order.
 
-2. Command 2 starts a Jupyter_ notebook server inside the
-docker container and forwards the port to `http://localhost:8888`.
-After running this command, you can copy-paste the complete URL from the terminal window where
-the command was run - to a browser to open a jupyter session to try out the included
-tutorials. Alternatively, you may simply point your browser to `http://localhost:8888`
-and, if prompted for a password, copy-paste the authentication token from the command
-window. Once successfully in the Jupyter notebook session, proceed to run the tutorials
-provided in the `examples` folder or create your own notebooks. 
+
+Docker for GPUs
+```````````````
+The GPU image differs from the CPU image regarding the additional compilers and toolkits necessary to run on GPUs. It contains the same installation and usage of Devito as the CPU image.
+
+Available images:
+
+- gpu-dev: for the latest development version (current github master)
+- gpu-latest: for the latest github release
+- gpu-vX.X.X: for a specific release (Release tags: https://github.com/devitocodes/devito/tags)
+
+Tags for several devito releases can be found here: https://github.com/devitocodes/devito/tags
+
+NOTE: To run the GPU version docker it is necessary to first install the `Nvidia container toolkit <https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#setting-up-nvidia-container-toolkit>`_.
+
+.. code-block:: shell
+
+   # NOTE (admin-sudo access may be needed) for some steps
```

Comment:
this sudo thing again...

Reply:
maybe check out other popular packages such as TensorFlow, or PyTorch, or whatever, and see what they write in terms of installation. Get inspired by what other smart people have already accomplished...